output/cloud/expv2/output.go

```diff
@@ -133,6 +135,16 @@ func (o *Output) Start() error {
 			TLSConfig: insights.ClientTLSConfig{
 				Insecure: false,
 			},
+			RetryConfig: insights.ClientRetryConfig{
+				RetryableStatusCodes: "UNKNOWN,INTERNAL,UNAVAILABLE,DEADLINE_EXCEEDED",
+				MaxAttempts:          3,
+				PerRetryTimeout:      30 * time.Second,
+				BackoffConfig: insights.ClientBackoffConfig{
+					Enabled:        true,
+					JitterFraction: 0.1,
+					WaitBetween:    1 * time.Second,
+				},
+			},
```

Comment:
Are those supposed to be configurable by the user?

Reply:
No, for them, these values won't make sense.